installers/vagrant-dev/radiasoft-download.sh

```diff
@@ -123,7 +122,7 @@ vagrant_dev_main() {
     vagrant_dev_modifiers
     for a in "$@"; do
         case $a in
-            fedora*|centos*)
+            fedora*|centos*|alma*)
```

Comment:
May want to do more syntax checking and call this $box not $os. There is code in vagrant_dev_eth1 that should be abstracted (e.g. `$(vagrant_dev_is_fedora $box)`). 

The name should be almalinux, not alma. We don't want to create our own name. There's a bug on line 27 where it is alma, but should be almalinux.

Reply:
d0e3a95